discordbot/cogs/internet.py

```diff
@@ -39,30 +53,35 @@ def __init__(self, bot: DiscordBot):
             user = user.display_name
         else:
             user = ctx.message.author.display_name
-        await ctx.send("https://robohash.org/{}.png".format(user.replace(" ", "%20")))
+        k = "https://robohash.org/{}.png".format(user.replace(" ", "%20"))
+        o = wget.download(k)
+        await ctx.send(file=o)
 
     @commands.command()
-    async def xkcd(self, ctx, query=""):
```

Comment:
This code is extremely old, and was planned to be pushed mere minutes before you opened this PR. I plan to push my entire working tree within the next few days, so it would be safer to open a PR then.